plugin/fubitive.vim

```diff
@@ -31,9 +31,12 @@ function! s:bitbucket_url(opts, ...) abort
   if protocol->empty()
       let protocol = 'https://'
   endif
+  let bitbucker_context_path = exists('g:fubitive_domain_context_path') 
```

Comment:
I'd prefer this:
```
let l:bitbucket_context_path = get( g:, 'fubitive_domain_context_path', '' )
```

And maybe we should define that variable at line 80, after the `if` and then just use it here.